### ISSUE TITLE
Crash when loaded alongside Deeper and Darker

### ISSUE THREAD

**Issue by ammoore00** [TheComputerizer/The-Impossible-Library#4]
Crash at startup in 1.19.2 forge when loaded alongside Deeper and Darker (https://www.curseforge.com/minecraft/mc-mods/deeperdarker). Not sure if that's something you are interested in dealing with but I figured I would report anyway just in case.

Unfortunately, any evidence of the crash was incredibly sparse. No crash report, no evidence of error in log or debug.log. Only thing I could find was in the launcher.log

Relevant error seems to be this line:

```Info Exception in thread "main" java.lang.module.ResolutionException: Modules theimpossiblelibrary and toml4j export package com.moandjiezana.toml to module nebs```

Whole error section provided for completeness

```[Info: 2023-01-06 03:59:21.4700522: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info Exception in thread "main" java.lang.module.ResolutionException: Modules theimpossiblelibrary and toml4j export package com.moandjiezana.toml to module nebs
[Info: 2023-01-06 03:59:21.4701670: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:901)
[Info: 2023-01-06 03:59:21.4702164: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
[Info: 2023-01-06 03:59:21.4702468: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
[Info: 2023-01-06 03:59:21.4702659: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
[Info: 2023-01-06 03:59:21.4702954: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
[Info: 2023-01-06 03:59:21.4703305: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
[Info: 2023-01-06 03:59:21.4703477: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:75)
[Info: 2023-01-06 03:59:21.4703632: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
[Info: 2023-01-06 03:59:21.4703794: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.run(Launcher.java:104)
[Info: 2023-01-06 03:59:21.4703937: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
[Info: 2023-01-06 03:59:21.4704071: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
[Info: 2023-01-06 03:59:21.4704213: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
[Info: 2023-01-06 03:59:21.4704348: MinecraftJavaLoggingContext.cpp(52)] Game/game () Info at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)```

**Comment by TheComputerizer** [comment]
Looks like the other mod is shading the toml reader dependency, thanks for the report I can fix it on my end

**Comment by TheComputerizer** [comment]
This has been fixed in 0.2.2
